### PR TITLE
Informal Update

### PR DESCRIPTION
**Pull request by frozenhelium**

### CODE REVIEW

File: src/root/views/InformalUpdateReport/index.tsx [hunk 1]
```diff
@@ -8,15 +8,18 @@ import Container from '#components/Container';
...
 
 import ViewSection from './ViewSection';
 import {
   ActionsTaken,
   InformalUpdateAPIFields
 } from '../InformalUpdateApplicationForm/common';
+
 import styles from './styles.module.scss';
+import Button from '#components/Button';
+import { IoShareSocialOutline } from 'react-icons/io5';
```
Comment (barshathakuri) [inline src/root/views/InformalUpdateReport/index.tsx:22]: Please move absolute imports before relative.

File: src/root/views/InformalUpdateReport/index.tsx [hunk 2]
```diff
@@ -34,42 +38,28 @@ function InformalUpdateReport(props: Props) {
...
-    // {link: this.props.location.state, name: strings.breadCrumbEmergency},
-    { link: '/Pakistan', name: breadCrumbReportTitle },
-    { link: '/emergencies', name: strings.breadCrumbEmergencies },
-    { link: '/', name: strings.breadCrumbHome }
-  ];
-
   const {
     pending: projectPending,
-    response: DataResponse,
+    response: reportResponse
```
Comment (AdityaKhatri) [inline src/root/views/InformalUpdateReport/index.tsx:42]:
```suggestion
    response: reportResponse,
```

File: src/root/views/InformalUpdateReport/index.tsx [hunk 3]
```diff
@@ -34,42 +38,28 @@ function InformalUpdateReport(props: Props) {
...
-    onSuccess: (response) => {
-      setInformalUpdateData(response);
-    }
   });
 
+  const crumbs = React.useMemo(() => [
+    { link: strings.informalUpdateNumber, name: `${strings.informalUpdateNumber}${reportResponse && reportResponse.id + 1}` },
+    { link: location?.pathname, name: reportResponse?.title },
+    { link: '/emergencies', name: strings.breadCrumbEmergencies },
+    { link: '/', name: strings.breadCrumbHome }
```
Comment (AdityaKhatri) [inline src/root/views/InformalUpdateReport/index.tsx:52]: break into multiple lines with each line having only one item

Comment (AdityaKhatri) [reply src/root/views/InformalUpdateReport/index.tsx:52]: Also, end the object with ,

File: src/root/views/InformalUpdateReport/index.tsx [hunk 4]
```diff
@@ -105,19 +95,17 @@ function InformalUpdateReport(props: Props) {
...
-          <span className='collecticon-chevron-right link--with-icon-inner'></span>
-        </>
+        title={reportResponse?.title}
+        heading={reportResponse?.title}
+        description={
+          <Link
+            to={location?.pathname}
+          >
+            <span className='link--with-icon-text'>
+              {reportResponse?.country_district?.map((item) => (item?.country_details?.name)).join('-')}
```
Comment (AdityaKhatri) [inline src/root/views/InformalUpdateReport/index.tsx:104]: Do this outside in a memo

File: src/root/views/InformalUpdateReport/index.tsx [hunk 5]
```diff
@@ -126,57 +114,74 @@ function InformalUpdateReport(props: Props) {
             compact={undefined}
           />
         }
+        actions={
+          <Button
+            variant='secondary'
+            name={undefined}
+            icons={<IoShareSocialOutline />}
```
Comment (AdityaKhatri) [inline src/root/views/InformalUpdateReport/index.tsx:120]: What does this button do? If the work is still left, add a TODO

File: src/root/views/InformalUpdateReport/index.tsx [hunk 6]
```diff
@@ -126,57 +114,74 @@ function InformalUpdateReport(props: Props) {
...
+                            {item.name}
+                          </div>
+                          <div className={styles.overviewValue}>
+                            {item.value}
+                          </div>
+                        </div>
+                        <div className={styles.overviewDescription}>
+                          {item.description}
+                        </div>
+                      </div>
```
Comment (AdityaKhatri) [inline src/root/views/InformalUpdateReport/index.tsx:146]: use ListView if possible, this is a big chunk and would be more readable if done in a separate component

File: src/root/views/InformalUpdateReport/index.tsx [hunk 7]
```diff
@@ -34,42 +38,28 @@ function InformalUpdateReport(props: Props) {
...
+    { link: '/', name: strings.breadCrumbHome }
+  ], [strings, location, reportResponse]);
+
   const renderActionTaken = (org: string) => {
-    const actions: ActionsTaken | undefined = informalUpdateData?.actions_taken.find(d => d.organization === org);
+    const actions: ActionsTaken | undefined = reportResponse?.actions_taken.find(d => d.organization === org);
     return (
       <ViewSection title='Actions taken by' data={actions} />
     );
   };
```
Comment (AdityaKhatri) [inline src/root/views/InformalUpdateReport/index.tsx:60]: Move this outside in a separate component

File: src/root/views/InformalUpdateReport/index.tsx [hunk 8]
```diff
@@ -126,57 +114,74 @@ function InformalUpdateReport(props: Props) {
...
+              >
+                <div className={styles.image}>
+                  {reportResponse?.graphics?.map((item) => (
+                    <div key={item?.id}>
+                      <img src={item?.file} alt="" />
+                    </div>
+                  ))}
+                </div>
+              </Container>
+              <Container heading={strings.informalUpdateActionTakenLabel} className={styles.actionTaken}>
```
Comment (AdityaKhatri) [inline src/root/views/InformalUpdateReport/index.tsx]: drop in multiple lines

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

**Pull request merged**